package.json

```diff
@@ -26,7 +26,7 @@
   },
   "peerDependencies": {
     "chai": "3.x",
-    "cheerio": "0.19.x || 0.20.x",
+    "cheerio": "0.22.0",
```

Comment:
Alternatively @ayrton @vesln, changing this to something like `0.x` could work - and then any reported breakages from cheerio, we could release a patch release with that version negated, e.g. if `0.23` broke, we just do `> 0.0.0 < 0.23.0 || > 0.23.0`